packages/core/src/odata/v4/request/odata-action-import-request-config.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { MapType } from '@sap-cloud-sdk/util';
+import { ODataUri } from '../../common/uri-conversion';
+import { FunctionImportParameters } from '../../common/request/function-import-parameter';
+import { ODataRequestConfig } from '../../common/request/odata-request-config';
+
+export class ODataActionImportRequestConfig<
+  ParametersT
+> extends ODataRequestConfig {
+  /**
+   * Creates an instance of ODataActionImportRequestConfig.
+   *
+   * @param defaultServicePath - Default path of the service
+   * @param actionImportName - The name of the function import.
```

Comment:
```suggestion
   * @param actionImportName - The name of the action import.
```
Maybe also check other places ;)